### PR TITLE
ONL-5446: feat: Pass the keys for what props should map the data, load…

### PR DESCRIPTION
**Pull request by jmesao**
Pass the keys for what props should map the data, loading and error

### CODE REVIEW

File: src/hocs/ec-with-abortable-fetch/ec-with-abortable-fetch.js
Comment (mcibique) [inline src/hocs/ec-with-abortable-fetch/ec-with-abortable-fetch.js:3]: We need to write a test for this.

**Review by mcibique — COMMENTED** [review]
There are 38 442 lines changed in package-lock.json. Please revert them as they are not related to this ticket.

File: src/hocs/ec-with-abortable-fetch/ec-with-abortable-fetch.js
Comment (jmesao) [reply src/hocs/ec-with-abortable-fetch/ec-with-abortable-fetch.js:3]: Done, many thanks!

**Comment by jaimeperezpedra** [comment]
I'll say normally we must wait until we have the approvals to bump the version of chameleon, otherwise, another PR can be merge before yours and get conflicts with the versions.

**Review by stavros-tomas — APPROVED** [review]

**Review by jaimeperezpedra — APPROVED** [review]

**Comment by stavros-tomas** [comment]
Good job Javi works on my local. Small note I think you bumped chameleon version on package.json manually. After you get Milos approval make sure you bump to correct version. We released  version 1.4.4 Friday.

**Comment by jmesao** [comment]
> Good job Javi works on my local. Small note I think you bumped chameleon version on package.json manually. After you get Milos approval make sure you bump to correct version. We released version 1.4.4 Friday.

Done man, thanks!

**Pull request merged**
